### DISCUSSION TITLE
Group bulbs together

### DISCUSSION THREAD

**Discussion by richl007** [sbidy/wiz_light#187] [Q&A] [answered]
Is there a way to sync 3 bulbs together? I want to be able to control the color and brightness of 3 bulbs all together.

**Comment by sjee** [comment, marked as answer]
You can create a light group. 

Once you made that light group you can control them via
- an "entity" card (but this will not show a switch to turn it on/off).
- an "entities" card and add the new light (which is actually the group of lights) and this will show an on/off switch.

Clicking on the name of the _light_ will bring up the colour and temperature picker.
Changes you make here will be applied to all the lights in the group.

More info, see: https://www.home-assistant.io/integrations/light.group/

_(I use an "entities" card to manually control the three Wiz lightbulbs in my [Ikea Simrishamn ](https://www.ikea.com/jo/en/p/simrishamn-floor-lamp-chrome-plated-opal-white-glass-50437768/) lamp.)_

**Comment by richl007** [comment]
Works, thank you!
